torch/_dynamo/optimizations/torchxla_integration/tests/reuse_graph_test.py

```diff
@@ -0,0 +1,127 @@
+import copy
```

Comment:
Let's put test files in the top level `test/dynamo` directory, with the right skips if XLA isn't installed.